.gitignore

```diff
@@ -4,3 +4,5 @@ yarn-error.log
 config.json
 /.vscode
 .idea
+package-lock.json
+package.json
```

Comment:
`package.json` shouldn't be ignored - it contains vital information about dependencies, scripts, license etc